src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.test.jsx

```diff
@@ -192,4 +201,198 @@ describe('LegacyConfigForm', () => {
         expect(container.querySelector(`#checkbox-${id}`)).toBeChecked();
       });
     });
+
+  test('Show help text on field focus', async () => {
+    await mockStore(legacyApiResponse);
+    createComponent(defaultAppConfig);
+    const button = getByTestId(container, 'add-topic');
+    userEvent.click(button);
```

Comment:
This will remain same everywhere, so move this to a method `"clickAddTopicButton(container)"`

Reply:
Or what you can do, is to create a method "createNewTopic(container, "topic name")`